### PR TITLE
Cleanup diplomacy

### PR DESCRIPTION
**Pull request by Cjkjvfnby**
Formatting:
- add two empty line between classes
- remove unused parenthesis from if statment
- remove unused parenthesis used for line wrapping
- remove `pylint` comments disable warnings

Code:
- remove unused imports
- reuse local variables
- make log message more compact

### CODE REVIEW

File: default/AI/DiplomaticCorp.py [hunk 1]
```diff
@@ -4,63 +4,65 @@
...
         :return:
         """
-        print "Received diplomatic %s message from empire %s to empire %s" % (message.type, message.sender, message.recipient)
-        print "my empire id: %s" % fo.empireID()
+        print "Received diplomatic %s message from %s to %s." % (
+            message.type, fo.getEmpire(message.sender),
+            'me' if message.recipient == fo.empireID() else fo.getEmpire(message.recipient))
         if message.recipient != fo.empireID():
             return
         reply_sender = message.recipient
```
Comment (Dilvish-fo) [inline default/AI/DiplomaticCorp.py]: regarding reply_recipient, message.sender, etc.,  consolidating terms makes plenty of sense, but now that the method is not only about making a reply, I think it better to just stick with message.sender, message.recipient terms.  We can  drop the use of reply_recipient and reply_sender.

**Comment by Dilvish-fo** [comment]
Note: I just put in a couple AI chat fixes & as part of that I clarified some variable names (regarding whether they refer to player IDs or empire IDs) and also took out the reply_recipient and reply_sender variables.

**Comment by Dilvish-fo** [comment]
ok, the current state of it looks fine.  In one of the comments you mentioned squashing it all into one commit; I think it is ready for that.   IIRC, you are using mostly pycharm and command line; does that leave you needing to redo the conflict resolution when you squash these commits?

**Comment by Cjkjvfnby** [comment]
I use `git merge --squash` and `push --force`

**Pull request merged**